examples/server/public/index.html

```diff
@@ -674,125 +756,145 @@
       const UserTemplateResetButton = () => {
         if (selectedUserTemplate.value.name == 'default') {
           return html`
-            <button disabled>Using default template</button>
-          `
+          <button disabled>Using default template</button>
+        `
         }
 
         return html`
-          <button onclick=${userTemplateReset}>Reset all to default</button>
-        `
+        <button onclick=${userTemplateReset}>Reset all to default</button>
+      `
       };
 
       useEffect(() => {
         // autosave template on every change
         userTemplateAutosave()
-      }, [session.value, params.value])
+      }, [session.value])// removed params.value as it would re render the page.
 
       const GrammarControl = () => (
         html`
-          <div>
-            <label for="template">Grammar</label>
-            <textarea id="grammar" name="grammar" placeholder="Use gbnf or JSON Schema+convert" value="${params.value.grammar}" rows=4 oninput=${updateParams}/>
-            <input type="text" name="prop-order" placeholder="order: prop1,prop2,prop3" oninput=${updateGrammarJsonSchemaPropOrder} />
-            <button type="button" onclick=${convertJSONSchemaGrammar}>Convert JSON Schema</button>
-          </div>
-          `
+        <details>
+        <summary>Grammer Configuration</summary>
```

Comment:
```suggestion
        <summary>Grammar Configuration</summary>
```

Reply:
Fixed all in new commit, Thanks for Review.